cdap-docs/developers-manual/source/conf.py

```diff
@@ -21,16 +21,23 @@ def setup(app):
     app.add_config_value('snapshot_version', '', 'env')
 
 # Set the condition
+archetype_version = "-DarchetypeVersion=%s" % release
 if release.endswith('SNAPSHOT'):
     snapshot_version = True
-    archetype_repository_version = ('-DarchetypeRepository=https://oss.sonatype.org/content/repositories/snapshots/' 
-      + " -DarchetypeVersion=%s" % release)
+    archetype_repository = '-DarchetypeRepository=https://oss.sonatype.org/content/repositories/snapshots/'
+    archetype_repository_version = "%s %s" % (archetype_repository, archetype_version)
 else:
     snapshot_version = False
-    archetype_repository_version = "-DarchetypeVersion=%s" % release
-    
+    archetype_repository = '-DarchetypeRepository=https://oss.sonatype.org/service/local/staging/deploy/maven2'
```

Comment:
Actually, now that I think about it...let me check with Terence and see if that is the correct value.